src/main/kotlin/org/cascadebot/cascadebot/data/objects/GuildSettingsCore.kt

```diff
@@ -105,10 +114,12 @@ class GuildSettingsCore {
     }
 
     fun addAlias(command: MainCommand, alias: String): Boolean {
+        if (!writeMode) throw UnsupportedOperationException("Cannot modify Guild data if not in write mode!")
```

Comment:
We should probably make a util function to do this, it's a lot of repeated code. `assertWriteMode()` perhaps?